packages/loader/container-loader/src/container.ts

```diff
@@ -2469,12 +2508,12 @@ export class Container
 			(this.protocolHandler.quorum.get("code") ??
 				this.protocolHandler.quorum.get("code2")) as IFluidCodeDetails | undefined;
 
-		const existing = snapshot !== undefined;
+		const existing = snapshotTree !== undefined;
```

Comment:
It should be always existing (has snapshot), no? It would be great cleanup types where possible from old crust

Reply:
yeah